bpf/cpu/cpu.bpf.c

```diff
@@ -747,15 +815,28 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
       break;
     }
 
+    //LOG("[debug] Switching to mixed-mode unwinding? %d", unwind_state->jitted);
```

Comment:
I think this could be a nice comment to have a log when we switch from the FP -> DWARF unwinder!

Reply:
I am not sure I understand this one. There already is a similar comment a few lines below when we switch
` // LOG("[debug] Switched to mixed-mode DWARF unwinding? %d", unwind_state->unwinding_jit);`
Do you want me to uncomment it? Or just add a comment instead of enclosing it in a log?

Reply:
Let's uncomment it!